ecommerce/src/main/kotlin/io/hhplus/ecommerce/common/consumer/DataPlatformConsumer.kt

```
@@ -0,0 +1,106 @@
+            } else {
+                // 실패 시 커밋하지 않음 → Kafka가 재전달
+                logger.warn(
+                    "[DataPlatformConsumer] 전송 실패, 재처리 예정: orderId={}, message={}",
+                    orderInfo.orderId, response.message
+                )
+                // 실패 시에도 ACK를 하지 않으면 무한 재시도가 될 수 있음
+                // 실무에서는 재시도 횟수 제한 + DLQ 이동 필요
+                acknowledgment.acknowledge()
+            }
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)